### ISSUE TITLE
[BUG] session_start_time is null

### ISSUE THREAD

**Issue by tahmid-H** [theOehrly/Fast-F1#103]
### Describe the issue:

I tried to get session_start_time for the 2021 season, but it's always returning null for any of the sessions. I tried grabbing the data by the Weekend, and also directly by Session. Same result - null. I've attached the code I'm using to get the weekend/session data.
All the other data are fine (date, location, session type), but it is just the session start time that is not working.

Edit: tested t0_date on 2019 Abu Dhabi Race, that also is null. 
### Reproduce the code example:

```python
# python
# for 2019 - FP1 Session
weekendData = ff1.core.get_session(2019, 2, 'FP1')
return {'data': weekendData.session_start_time}

# for 2021 Abu Dhabi - Race Session
weekendData = ff1.core.get_session(2019, 'Abu Dhabi', 'R')
return {'data': weekendData.session_start_time}

# for 2021 Abu Dhabi - Whole Weekend
weekendData = ff1.core.get_session(2019, 'Abu Dhabi')
race = weekendData.get_race()
return {'data': race.session_start_time}
```

### Error message:

```shell
{
   data : null
}
```

**Comment by theOehrly** [comment]
`session_start_time` is the actual time the session was started and not some scheduled starting time. It is therefore extracted from the session status data which is only loaded once `Session.load_laps` is called. Sorry, this is not documented really and rather unclear. I'm aiming to improve how this work in a future version of FastF1.

TL;DR: you currently need to call `Session.load_laps` to load this value.

**Comment by theOehrly** [comment]
I'm currently working on implementing a proper event schedule, which will provide scheduled session start times for each weekend. That should be what you are looking for? This will be part of FastF1 v2.2 which will be released before the first race (hopefully...). Maybe I'll provide some beta releases before that. 

Also, most of the 2022 sessions have official starting times since one or two days ago.

**Comment by tahmid-H** [comment]
Yes! If you need a beta tester, I volunteer! Thank you for this amazing API.

Also, I just realized they released the timings. I tried grabbing the timing from api.formula1.com/v1/event-tracker, but they only gave me the time for Bahrain. I'm probably doing something wrong, so I'll research in the mean time until your API gets the new function!

**Comment by theOehrly** [comment]
Take a look at https://github.com/theOehrly/f1schedule
This should contain all the current official starting times. You can compare it with the schedule in formula1.com . It's usually updated regularly. But I can't give any guarantees in terms of "api" consistency. This is very much alpha grade stuff. But it'll be the "backend" for FastF1's event schedule for now. The json file is formatted so that it can be read directly with `pandas.read_json`

**Comment by tahmid-H** [comment]
Perfect! Thank you so much!

**Comment by tahmid-H** [comment]
Oo that's what I was missing. From inspecting the API call which the schedule page makes, I only found event-tracker. Looks like I didn't dig deep enough. Thank you for providing so much help for the F1 fans who like to get technical.

**Comment by theOehrly** [comment]
The first pre-release version for v2.2 is now available here https://github.com/theOehrly/Fast-F1/discussions/105

This release already includes the new event schedule. You can check it out if you're interested.

**Comment by Plus1XP** [comment]
Hi @theOehrly, can I ask how you found v1/editorial-eventlisting/events as the endpoint? Im trying to make a program to get event information from F1,MotoGP, WSBK etc. and im wondering what I done wrong as I could only find `/v1/event-tracker` just like @tahmid-H  Thanks
